src/sso/sso.ts

```diff
@@ -0,0 +1,27 @@
+import queryString from 'query-string';
+
+import { API_HOSTNAME } from '../common/constants';
+import { AuthorizeURLOptions } from './interfaces/authorize-url-options.interface';
+import WorkOS from '../workos';
+
+export class SSO {
+  constructor(private readonly workos: WorkOS) {}
+
+  getAuthorizeURL({
```

Comment:
`getAuthorizationURL`?

Reply:
I think I agree with Mark - "Authorization URL" feels like a standard term noun to me?

Reply:
Sweet, just updated it @marktran @sbauch